### ISSUE TITLE
No longer scraping content in messages

### ISSUE THREAD

**Issue by hydr0homiee** [DIGITALCRIMINALS/OnlyFans#587]
Previously I was able to scrape ~90% of content sent in DMs. Occasionally the scraper would miss some paid content sent in messages but still download the rest. I tried running the scraper today and noticed that the scraper only grabbed like 1/30 videos from a model's messages. Has anyone else noticed this?

When scraping, I scrape by individual model instead of using `0 = All`. I found scraping all would randomly delete scraped content (issue #447).

I'm running Windows 10, whatever version the main branch is (8.0?) as of [the patch on July 27th.](https://github.com/DIGITALCRIMINALS/OnlyFans/commit/cf723726ef1d174c629e785633b826ec9b173c7c). Not sure if this is an update on OF's end or something with the scraper. Here's my config.json.

```
{
  "info": {
    "version": 8.0
  },
  "settings": {
    "auto_site_choice": "1",
    "export_type": "json",
    "profile_directories": [
      ".profiles"
    ],
    "max_threads": -1,
    "min_drive_space": 0,
    "helpers": {
      "renamer": true,
      "reformat_media": true,
      "downloader": true,
      "delete_empty_directories": false
    },
    "webhooks": {
      "global_webhooks": [],
      "global_status": true,
      "auth_webhook": {
        "succeeded": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        },
        "failed": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        }
      },
      "download_webhook": {
        "succeeded": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        },
        "failed": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        }
      }
    },
    "exit_on_completion": false,
    "infinite_loop": true,
    "loop_timeout": "0",
    "dynamic_rules_link": "https://raw.githubusercontent.com/DATAHOARDERS/dynamic-rules/main/onlyfans.json",
    "proxies": [],
    "cert": "",
    "random_string": "49556f17ff1511eb86d86c5d3a14a5f6"
  },
  "supported": {
    "onlyfans": {
      "settings": {
        "auto_profile_choice": [1],
        "auto_model_choice": false,
        "auto_api_choice": true,
        "auto_media_choice": "0",
        "browser": {
          "auth": true
        },
        "jobs": {
          "scrape": {
            "subscriptions": true,
            "messages": false,
            "paid_content": false
          },
          "metadata": {
            "posts": true,
            "comments": true
          }
        },
        "download_directories": [
          ".sites"
        ],
        "file_directory_format": "{site_name}/{model_username}/{media_type}",
        "filename_format": "{filename}.{ext}",
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}/{model_username}/Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [ ],
        "ignore_type": "",
        "blacklists": [],
        "webhook": true
      }
    },
```

**Comment by left1000** [comment]
I had a similar issue occur for me. Not only did it delete everything but it even said my date of folder creation was today, making me think I'd never made the rip in the first place. Thankfully I had a backup.

Going to stick to 7.6.1 as the issue never occurs for me on 7.6.1 and occurred for me the first time I tried running the current update/commit.

I actually was and only ever have run individual model scrapes. After I ran the current commit/update for one model it just moved to the next model, and when I selected a media type, it just nuked that model's entire prior scrape/rip.

Thankfully I noticed after 2 models, and 1 was backed up and 1 I'd never ripped anyways.
